applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/eigensolver_strategy.hpp

```diff
@@ -402,6 +402,9 @@ class EigensolverStrategy
 
         rModelPart.GetProcessInfo()[BUILD_LEVEL] = 1;
         TSparseSpace::SetToZero(rMassMatrix);
+
+        pScheme->InitializeNonLinIteration(rModelPart, rMassMatrix, b, b); // arguments except ModelPart are unused in Scheme.
```

Comment:
Even it is not used, it seems more natural to me to pass the `rStiffnessMatrix`. Up to you